builds/e2e/templates/longhaul-deploy.yaml

```diff
@@ -4,6 +4,7 @@ parameters:
   test.buildId: ''
```

Comment:
Can you change the other reference to Build.SourcesDirectory in this file (line 94) to be the parameter. It's a bit confusing otherwise to use both

Reply:
and 96

Reply:
yeah good catch

Reply:
I also updated the connectivity tests to read from this checkout path